scripts/vulnerabilities.sh

```diff
@@ -50,39 +42,34 @@ find_askpass_image_tag() {
 
 GCENODE_ASKPASS_TAG="${GCENODE_ASKPASS_TAG:-$(find_askpass_image_tag)}"
 
-# TODO: share with Makefile to keep in sync
-configsync_images=(
-  "gcr.io/${GCP_PROJECT}/reconciler"
-  "gcr.io/${GCP_PROJECT}/reconciler-manager"
-  "gcr.io/${GCP_PROJECT}/admission-webhook"
-  "gcr.io/${GCP_PROJECT}/hydration-controller"
-  "gcr.io/${GCP_PROJECT}/hydration-controller-with-shell"
-  "gcr.io/${GCP_PROJECT}/oci-sync"
-  "gcr.io/${GCP_PROJECT}/helm-sync"
-  "gcr.io/${GCP_PROJECT}/nomos"
-)
-
 # Build a full list of images with tags
 images=(
-  "gcr.io/${GCP_PROJECT}/gcenode-askpass-sidecar:${GCENODE_ASKPASS_TAG}"
```

Comment:
Why hardcode the project? Don't we want to support deploying to other projects?

Reply:
This is released from a different project and should be expected to be in this registry. The actual name of this registry isn't from `GCP_PROJECT` but rather whatever value is hardcoded [here](https://github.com/GoogleContainerTools/kpt-config-sync/blob/main/pkg/reconcilermanager/controllers/gcenode_askpass_sidecar.go#L47).

This is just a hack to get around the fact that we declare this image in code rather than the manifests. The real solution would be to manage the image in manifests, IMO.